theme/templates/resource-landing-page/title-section.html

```diff
@@ -154,6 +154,41 @@
             </div>
         </div>
     {% endif %}
+    <div id="checking-for-paths" class="col-sm-12">
+        <p class="alert alert-warning">Checking for non-preferred file/folder path names ...</p>
```

Comment:
we might have to add a note of some sort letting the user know that this check can take a long time...

Reply:
Good idea. Added the needed to text.